week1/3-website/index.html

```diff
@@ -16,7 +17,90 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header id="header">
+        
+        <h1>The importance of good sleep</h1>
+        <!--<img id="header-img" src="images/sleepingCat.jpeg"      height: 90px;    width: 50px;  alt="a sleeping cat" />-->
```

Comment:
You can't promise a cat image and then not deliver it.

Reply:
In this case, it would work if you replaced 'images/sleepingCat.jpeg' with 'sleepingCat.jpeg' assuming the image is correctly encoded.